client/blocks/image-editor/index.jsx

```diff
@@ -103,6 +108,16 @@ const ImageEditor = React.createClass( {
 		this.setDefaultAspectRatio();
 	},
 
+	componentWillMount() {
+		const {
+			media
+		} = this.props;
+
+		this.props.setImageMeetsMinimumDimensions(
+			meetsMinimumDimensions( media.width, media.height )
+		);
```

Comment:
We can remove this, but this feels a bit backwards. Unless it's a user event, we can normally compute this in the reduce from dispatched events.